RInAppMessaging/Classes/Extensions/CGSize+IAM.swift

```diff
@@ -0,0 +1,6 @@
+extension CGSize {
+    /// returns a `CGSize` with the smallest integer values for its size to prevent drawing on pixel boundaries
+    var integral: CGSize {
+        CGSize(width: ceil(width), height: ceil(height))
```

Comment:
ceil fixes this issue because width and height can contain decimals?

Reply:
Yes, that seemed like it was the issue